components/link.tsx

```diff
@@ -0,0 +1,13 @@
+import { AllHTMLAttributes } from 'react';
```

Comment:
Removed imported `HTMLAttributes` since it is never used, and changed to relative imports